Add FAQ to README.md (#35)
paypal/iOS-SDK: README.md

Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payment methods in your iOS app.

## FAQ
### Availability

> The SDK is currently in the development process. This product will being developed fully open source - throughout the development process, we welcome any and all feedback. We recommend using the SDK in the sandbox environment until an official general availability release is out as aspects of the SDK _will likely_ change as we develop the SDK. This README will be updated with an official release date as it becomes available.

Reviewer: ```suggestion
The SDK is currently in the development process. This product is being developed fully open source - throughout the development process, we welcome any and all feedback. Aspects of the SDK _will likely_ change as we develop the SDK. We recommend using the SDK in the sandbox environment until an official release is available. This README will be updated with an official release date once it is generally available.
```

Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payment methods in your iOS app.

## FAQ
### Availability
The SDK is currently in the development process. This product will being developed fully open source - throughout the development process, we welcome any and all feedback. We recommend using the SDK in the sandbox environment until an official general availability release is out as aspects of the SDK _will likely_ change as we develop the SDK. This README will be updated with an official release date as it becomes available.

### Contribution

> As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue/PR with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.

Reviewer 2: ```suggestion
As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
```

Do we want people to use the GH discussions feature? It's kind of nice for sharing ideas

Author: Yeah - we should for sure use the discussions feature for this!

> 

Reviewer 2: ```suggestion
```
